hardware/fpga/QUAD_CORE.md

```diff
@@ -0,0 +1,61 @@
+# Step-by-step 4-core bitstream generation
+
+```
+git clone git@github.com:AlSaqr-platform/he-soc.git
+cd he-soc
+git checkout alsaqr-4-corev1.0
+```
+
+Change the `PATH` and `RISCV` exports in the `setup.sh` script.
+
+```
+source setup.sh
+cd hardware
+make bender update
+make simple-padframe=1 exclude-cluster=1 exclude-rot=1 quad-core=1 scripts-bender-fpga-ddr
+cd fpga
+source setup.sh
+```
+Correct setup settings are highlighted in bold:
+
```

Comment:
Change this README according to the updated `setup.sh`.

Reply:
This issue was addressed in https://github.com/AlSaqr-platform/he-soc/pull/118/commits/7167a1181efe343bd11fdb947303796ae52c6507.